src/index.html

```diff
@@ -11,8 +11,55 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image"></div>
+
+      <h1 class="card__product-name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
```

Comment:
```suggestion
      <h2 class="card__product-name">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </h2>
```
It's expected to have multiple cards on the page. However, the `h1` tag should be used just once per page